Gruntfile.js

```diff
@@ -1,4 +1,11 @@
 module.exports = function(grunt) {
+  var account;
+  try {
+    account = require('add');
+  } catch (e) {
+    account = {email: false, password: false};
+  }
```

Comment:
I'm using the environment variables for it. I'm also fine adding another method, could you quickly add it to the README upload section how to configure?